src/backend/AuthService/BusinessLogic/PostLogin/PostLoginAction.cs

```diff
@@ -34,7 +34,7 @@ public async Task<ActionResult> Execute()
 
             if (!isUserInfoValid)
             {
-                throw new PostLoginDataCheckValidationException("CurrentAuth0UserInfo in PostLoginAction didn't pass validation");
+                throw new CurrentUserDataCheckValidationException("CurrentAuth0UserInfo in PostLoginAction didn't pass validation");
```

Comment:
Ostatnio puściłem na to oko, ale to jest coś co powinno zostać poprawione.

`throw new MyException1(....)` zwróci wiadomość dla użytkownika końcowego, który jest nietechniczny. To co tutaj napisałeś jest dla niego totalnie bezużyteczne, ale z kolei dla administratora systemu coś tam podpowiada.

Jeśli chcesz to zrobić profesjonalnie, powinieneś zrobić tak:
1. Zmienić nazwę wyjątku `PostLoginDataCheckValidationException` na `InvalidAuth0DataException` bez parametrów. W tej klasie możesz napisać na sztywno informację, że coś jest nie tak z danymi Auth0 i użytkownik powinien skontaktować się z administratorem serwisu.
2. Zinjectować klasę `ILogger<AktualnaKlasa> _logger`, w tym przypadku `ILogger<PostLoginAction>`. Jak będzie ci się to sypało, to odezwij się do mnie, być może będę musiał to skonfigurować.
3. Przed zwracaniem wyjątku, wykonać `_logger.LogError(...)`, gdzie bardzo dokładnie opiszesz problem, np: "Obiekt X nie przeszedł walidacji, bo znaleziono nulla". Taka informacja wyświetli się tylko administratorowi systemu i będzie dla niego bardzo cenna.

Reply:
Przetestowałem i działa:
![image](https://user-images.githubusercontent.com/32263891/215263675-50d9313b-36d5-4584-8c7f-313719886a32.png)

Reply:
Chyba git